src/sentry/shared_integrations/client.py

```diff
@@ -62,14 +62,14 @@ def from_response(self, response, allow_text=False):
 
         # Some APIs will return JSON with an invalid content-type, so we try
         # to decode it anyways
-        if "application/json" not in response.headers["Content-Type"]:
+        if "application/json" not in response.headers.get("Content-Type", {}):
             try:
                 data = json.loads(response.text, object_pairs_hook=OrderedDict)
             except (TypeError, ValueError):
                 if allow_text:
                     return TextApiResponse(response.text, response.headers, response.status_code)
                 raise UnsupportedResponseType(
-                    response.headers["Content-Type"], response.status_code
+                    response.headers.get("Content-Type", "No Content-Type"), response.status_code
```

Comment:
I don’t know the context but default should be empty string. This could be confusing since “No Content-Type” it’s was the header is.

Reply:
@mattrobenolt Yea, makes sense. I'll make this change.